.github/workflows/test-and-build.yaml

```diff
@@ -1,12 +1,12 @@
-name: make test
+name: make test and docker-build
```

Comment:
as we are running only docker build change this do make docker-build and also jobs name need to be updated

Reply:
Actually this will run test also as part of build 

https://github.com/kube-storage/volume-replication-operator/blob/main/Makefile#L94

Reply:
Agree but that is not the action command we ran right? it might be doing it internally.

Reply:
Did it in a different way